### PR TITLE
Update/add more tools

### PR DESCRIPTION
**Pull request by moon0326**
This PR refactors the existing Tools section to add more commands (not sure if this is the right terminology or not).

This PR also makes it easy to add more commands. Refer to [README](https://github.com/woocommerce/woocommerce-admin-test-helper/blob/52f04f5edb52abb9c7fca5f68b6faaed4db5775a/src/tools/README.md) on how to add a new command.

I've added the following commands in this PR, but more are coming.

- Reset Onboarding Wizard
- Reset Jetpack Connection
- Enable wc-admin* event tracking debug mode
- Update WC Installation timestamp

## Screenshots

Screen Shot 2021-04-28 at 4 53 36 PM

### CODE REVIEW

File: api/options/rest-api.php [hunk 1]
```diff
@@ -28,23 +28,27 @@
...
 			),
 		),
 	)
 );
 
 function wca_test_helper_delete_option( $request ) {
 	global $wpdb;
-	$option_id = $request->get_param( 'option_id' );
-	$query     = $wpdb->prepare( "delete from {$wpdb->prefix}options where option_id = %d", $option_id );
+	$option_names = explode(',', $request->get_param( 'option_names' ));
```
Comment (becdetat) [inline api/options/rest-api.php:45]: We don't have any linting, but we should probably stick with WP conventions for spacing around calls etc.

Comment (becdetat) [reply api/options/rest-api.php]: Ignore this, I tweaked it myself (hope that's ok :-) )

**Review by becdetat — APPROVED** [review]
Looks great @moon0326 :-D. Ignore my comment, I've added a couple of commits fixing a typo and some spacing. Hope that's ok with you.

**Comment by moon0326** [comment]
> Looks great @moon0326 :-D. Ignore my comment, I've added a couple of commits fixing a typo and some spacing. Hope that's ok with you.

Thank you for the fixes 🙇‍♂️ 

I should add auto fix to my IDE :)

**Pull request merged**